src/nextjs/server/index.tsx

```diff
@@ -111,7 +113,7 @@ export function isAuthenticatedNextjs() {
  */
 export type ConvexAuthNextjsMiddlewareContext = {
   getToken: () => string | undefined;
-  isAuthenticated: () => boolean;
+  isAuthenticated: () => Promise<boolean>;
```

Comment:
You probably need a rebase since I think these are already async https://github.com/get-convex/convex-auth/pull/125

Reply:
👍